rbc/heavydb/timestamp.py

```diff
@@ -1,17 +1,258 @@
 """HeavyDB Timestamp type that corresponds to HeavyDB type timestamp."""
```

Comment:
Very minor but shouldn't this read `RBC Timestamp type that corresponds...`? As it is it's basically a tautology :p

Reply:
Thanks!